matter_server/common/helpers/api.py

```diff
@@ -44,8 +42,12 @@ def get_typed_signature(call: Callable) -> inspect.Signature:
     return signature
 
 
-def parse_arguments(func_sig: inspect.Signature, args: dict, strict: bool = False):
+def parse_arguments(
+    func_sig: inspect.Signature, args: Optional[dict], strict: bool = False
```

Comment:
I included this fix to make the tests pass when passing `None` for `args`, as we also do sometimes in the server.

I can break this commit out if wanted.

Reply:
its fine like this, keep it this way. In the long run I want to have a second look at that whole argument parsing thingie but not for today